internal/rpc/clients.go

```diff
@@ -39,7 +40,8 @@ import (
 )
 
 const (
-	configNameEnableRPCLogging = "logging.rpc"
+	// ConfigNameEnableRPCLogging is the config name for enabling RPC logging.
```

Comment:
Could we have an internal/rpc/const.go file for these package level constants?

Reply:
I'll do that in my follow up change.